### PR TITLE
LRRZ new RCET station

### PR DESCRIPTION
**Pull request by mnaguit**
- for LC 10 (LH/HH-ENZ) streams
- Centaur (SN 7983) firmware v4.5.20

### CODE REVIEW

**Review by salichon — APPROVED** [review]
new station Rcet RA 
sweet

**Review by CBurton90 — APPROVED** [review]

**Comment by mnaguit** [comment]
Ready to merge this one, once @ozym approves 😄

**Review by ozym — APPROVED** [review]

**Pull request merged**
